aexpect/remote.py

```diff
@@ -11,15 +11,8 @@
 import tempfile
 
 import aexpect
-from avocado.core import exceptions
-from avocado.utils import process
-
-from virttest import data_dir
 from virttest import utils_misc
 from virttest import rss_client
-from virttest.remote_commander import remote_master
-from virttest.remote_commander import messenger
-from virttest.compat_52lts import results_stdout_52lts, results_stderr_52lts
```

Comment:
I don't use these and they are fairly advanced. I think they can stay in virttest only so yes to this commit.

Reply:
Indeed, in order for us to best utilize resources I would say that if someone is using them very intensely and they want them in aexpect, they can always raise an issue to request this or contribute with their own migration as they might already know this code better as well.